views/includes/piwik-scripts.html

```diff
@@ -1,6 +1,14 @@
 <script src="{{MATOMO_ASSET_PATH}}/javascripts/app/piwik-enable.js"></script>
 <script type="application/javascript">
-    window.SERVICE_NAME = 'Register of overseas entities'
+    const journeyTypes = {
+        "register-an-overseas-entity": "Register",
+        "update-an-overseas-entity": "Update"
+    };
+    
+    const uri = window.location.href;
+    const journeyFromUri = journeyTypes[uri.replace(/^(https?:\/\/)?/, '').split('/')[1]];
```

Comment:
rules out http:// but probably fine for that.  journeryFromUri = url.contains("//update-an-overseas-entity/")) ? "Update" : "Registration"; would probably have been fine?

Reply:
I think we wanted to future proof it so that if other journeys are added in the future (eg. Remove) its easy enough to add it in to the associative array